README.md

```diff
@@ -100,18 +99,18 @@ errors = {
     ],
 }
 ```
-Здесь `errors` это словарь, в котором ключи - путь к файлу, а значение этого ключа -
-список ошибок, где каждая ошибка - словарь
+Here `errors` is a dictionary, where the keys are the path to the file, 
```

Comment:
Here `errors` is a dictionary, where the keys are the path to the file, and the value of this key is a list of errors, where each is a dictionary.